plugin/src/main/java/io/spine/tools/gradle/config/ArtifactSnapshot.java

```diff
@@ -80,6 +84,8 @@ private static ArtifactSnapshot load() {
                 .setSpineCoreVersion(properties.getProperty("spine.version.core"))
```

Comment:
Please see if we can have a class which declares these constants and used at both ends.
If it's not easy, please document usage of this constants at both ends: when they generated, and here (consumed), referring to the other end of this trick.